README.md

```diff
@@ -761,6 +768,42 @@ test('Example test', async function(assert) {
 });
 ```
 
+#### Test helper option: `targetSelector`
+
+The selector of the target element of the tooltip or popover you are testing.
+
+If more than one tooltip or popover is found in the DOM with a particular selector
+when you run an assertion, you will be asked to specify this.
+
+| Type    | String |
+|---------|---------|
+| Default | `'.ember-tooltip-target, .ember-popover-target'` |
+
+Usage example:
+
+```js
+import { render, triggerEvent } from '@ember/test-helpers';
+import { assertTooltipVisible } from 'ember-tooltips/test-support';
+
+test('Example test', async function(assert) {
+
+  await render(hbs`
+    <div class="target-a">
+      {{ember-tooltip class="common-tooltip" side='top' isShown=true text='Hi' effect='none'}}
+    </div>
+    <div class="target-b">
+      {{ember-tooltip class="common-tooltip" side='left' isShown=true text='Bye' effect='none'}}
+    </div>
+  `);
+
+  await triggerEvent(this, this.element);
+
+  assertTooltipVisible(assert, {
+    selector: '.target-b', // Or whatever class you added to the target element
```

Comment:
Is this example for `selector` or `targetSelector`? It should be the latter.

Reply:
doh! yup, good catch!